README.md

```diff
@@ -10,7 +10,7 @@ Feature flippers.
 ## MAKE SURE TO READ THIS: 2.X Changes and Migration Path
 
 As of rollout-2.x, only one key is used per feature for performance reasons.
-The data format is `percentage|user_id,user_id,...|group,_group...`. This has
+The serialized format is `percentage|user_id,user_id,...|group,_group...|data_key_1=>data_val_1|data_key_2=>data_val_2|...`. This has
```

Comment:
is there particular reason why you chose your own made up format instead of something standard (eg. JSON)?